src/common/NetworkEventManager.ts

```diff
@@ -12,10 +12,11 @@ export type QueuedEvents = {
 export type FetchRequestId = string;
 export type NetworkRequestId = string;
 
-export type RedirectInfoMap = Array<{
+type RedirectInfo = {
   event: Protocol.Network.RequestWillBeSentEvent;
   fetchRequestId?: FetchRequestId;
-}>;
+};
+export type RedirectInfoMap = RedirectInfo[];
```

Comment:
If we are renaming types anyway, should we change the name of this type as well?
```suggestion
export type RedirectInfoList = RedirectInfo[];
```